catkit2/services/oceanoptics_spectrometer_sim/oceanoptics_spectrometer_sim.py

```diff
@@ -0,0 +1,156 @@
+'''
+This module contains a service for simulation Ocean Optics Spectrometers.
+'''
+import numpy as np
+from catkit2.testbed.service import Service
+
+
+class OceanOpticsSpectrometerSim(Service):
+    '''
+    Service for Ocean Optics Spectrometer.
+
+    This service is a wrapper around the python seabreeze package.
+    It provides a simple interface to control the spectrometer and acquire spectra.
+
+    Attributes
+    ----------
+    spectrometer : Spectrometer
+        The spectrometer to control.
+    serial_number : str
+        The serial number of the spectrometer.
+    model : str
```

Comment:
All the attributes present in the hardware service should still exist in the simulated service where possible. So please still read the serial number and model from the config in this simulated service as well. You can delete the spectrometer from the Attributes list.

Reply:
ok for serial_number. 

Model in not in the yaml, this is a spectrometer attribute that you can get from seabreeze. I thought it could be useful so I created a service attribute for it if someone want to use it wihtout having to parse the documentation of seabreeze package. However, I cannot have it in the simulated service